doofinder.php

```diff
@@ -45,15 +46,15 @@ class Doofinder extends Module
 
     const GS_SHORT_DESCRIPTION = 1;
     const GS_LONG_DESCRIPTION = 2;
-    const VERSION = '4.2.2';
+    const VERSION = '4.2.3';
     const YES = 1;
     const NO = 0;
 
     public function __construct()
     {
         $this->name = 'doofinder';
         $this->tab = 'search_filter';
-        $this->version = '4.2.2';
+        $this->version = '4.2.3';
```

Comment:
```suggestion
        $this->version = '4.3.0';
```
It is a new functionality. I would put this version